### PR TITLE
Add description for migration of old databases

### PR DESCRIPTION
**Pull request by obraliar**

### CODE REVIEW

**Comment by koppor** [comment]
Could you use "pre-3.6" instead of "old" (filenames, text)? "pre-3.6" is more exact.

File: en/migrateOldDatabase.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+title: Migration of pre-3.6 databases into shared database
+---
+
+# Migration of pre-3.6 databases into shared database
+
```
Comment (mlep) [inline en/migrateOldDatabase.md]: Mention in both both titles that this is for *SQL* database

File: en/migrateOldDatabase.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+![Screenshot of migration popup](./images/migrate-pre-3.6-db.png)
+
+## Migration
+
+To migrate your pre-3.6 database into new shared database you will have to follow these steps:
+
+-	Download and install [JabRef 3.5](https://github.com/JabRef/jabref/releases/tag/v3.5)
+-	Open JabRef and goto **File** -> **Import from external SQL database**
+-	Enter required data and klick on **Connect**
```
Comment (mlep) [inline en/migrateOldDatabase.md]: klick --> click

File: en/migrateOldDatabase.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+-	Choose the database which should be imported and press **Import**
+-	Save the database locally (**File** -> **Save database**)
+-	Turn back at least to [JabRef 3.6](https://github.com/JabRef/jabref/releases/tag/v3.6)
+-	Goto: **File** -> **Open shared database**
+-	Enter required data and klick on **Connect**
```
Comment (mlep) [inline en/migrateOldDatabase.md]: klick --> click

File: en/migrateOldDatabase.md (hunk 1 above)
Comment (obraliar) [inline en/migrateOldDatabase.md]: Done.

File: en/migrateOldDatabase.md (hunk 2 above)
Comment (obraliar) [inline en/migrateOldDatabase.md]: Done.

File: en/migrateOldDatabase.md (hunk 3 above)
Comment (obraliar) [inline en/migrateOldDatabase.md]: Done.

**Pull request merged**